proto/cmp/types/v3/currency.proto

```
@@ -0,0 +1,1312 @@
+syntax = "proto3";
+
+package cmp.types.v3;
+
+import "cmp/types/v2/token.proto";
+import "google/protobuf/empty.proto";
+
+// See extensive explanation at Price in price.proto
+message Currency {
+  oneof currency {
+    cmp.types.v2.IsoCurrency iso_currency = 1;
```

buf (1.47.2)

field cmp.types.v3.Currency.iso_currency: unknown type cmp.types.v2.IsoCurrency

(COMPILE)